pkg/kubelet/secret/secret_manager.go

```diff
@@ -209,8 +209,12 @@ type cachingSecretManager struct {
 }
 
 func NewCachingSecretManager(kubeClient clientset.Interface) (Manager, error) {
+	// TODO: Currently apiserver completely cannot keep up with 1 minute ttl in
+	// 5000-node clusters. So before we optimize apiserver and/or etcd, increase
+	// the TTL to 2 minutes (which is a lot, but...)
+	ttl := 2 * time.Minute
```

Comment:
@yujuhong @dchen1107 - can you suggest the best way for making this tunnable? From what I understand you're not a fan of adding a new flag. On the other hand, IIUC dynamic kubelet config still doesn't exist, and we need something short term.

So do you think that the following plan is reasonable:
- short term add a flag for it (with 1m default)
- the question remains how exactly we should be setting this flag...
- long term move it to dynamic kubelet config
?

One alternative I can see (which from implementation point of view might be even significantly easier) is to:
- right before creating a secret manager (or even in its constructor, for better encapsulation of that logic) LIST all nodes in the cluster
- based on the cluster size, set the ttl 

WDYT?